artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master/Middleware/ScrumProjectIntegrationMiddleware.cs

```diff
@@ -0,0 +1,38 @@
+using Artificial.Scrum.Master.ScrumProjectIntegration.Exceptions;
+
+namespace Artificial.Scrum.Master.Middleware
+{
+    public class ScrumProjectIntegrationMiddleware : IMiddleware
+    {
+        private readonly ILogger<ScrumProjectIntegrationMiddleware> _logger;
+
+        public ScrumProjectIntegrationMiddleware(ILogger<ScrumProjectIntegrationMiddleware> logger)
+        {
+            _logger = logger;
+        }
+
+        public async Task InvokeAsync(HttpContext context, RequestDelegate next)
```

Comment:
Nie wierze że po stronie frontu będziemy to obsługiwać ale ok fajnie że o tym pomyślałeś

Reply:
Tylko nie wiem czy exception middleware powinno być w module który nie tak naprawde nie powinien wiedzieć o asp.net, (wiem że są endpointy ale nie chciało mi się rozbijać na kilka projektów)